src/libs/ajax/User.ts

```diff
@@ -68,46 +92,63 @@ export const User = {
     }
   },
 
-  registerUser: async () => {
+  registerUser: async (): Promise<DuosUser> => {
     const url = `${await getApiUrl()}/api/user`;
-    const res = await fetchOk(url, fp.mergeAll([Config.authOpts(), { method: 'POST' }]));
+    const res = await fetchOk(
+      url,
+      fp.mergeAll([Config.authOpts(), { method: 'POST' }])
+    );
     return res.json();
   },
 
-  getSOsForCurrentUser: async () => {
+  getSOsForCurrentUser: async (): Promise<SimplifiedDuosUser[]> => {
     const url = `${await getApiUrl()}/api/user/signing-officials`;
-    const res = await fetchOk(url, fp.mergeAll([Config.authOpts(), { method: 'GET' }]));
+    const res = await fetchOk(
+      url,
+      fp.mergeAll([Config.authOpts(), { method: 'GET' }])
+    );
     return res.json();
   },
 
-  getUnassignedUsers: async () => {
+  getUnassignedUsers: async (): Promise<DuosUserResponse> => {
```

Comment:
Should this be returning a list?
```suggestion
  getUnassignedUsers: async (): Promise<DuosUserResponse[]> => {
```

Reply:
yes, good catch 👍